packages/webapp/src/components/Map/SelectionHandler/index.jsx

```diff
@@ -50,22 +50,23 @@ export default function PureSelectionHandler({ locations, history, sensorReading
 
   const [isSensor, setIsSensor] = useState(false);
   const [sensorIdx, setSensorIdx] = useState(null);
+  const longPressed = useRef(false);
 
-  let longPressed, longPressTimeout, longPressActive;
-
-  const handleMouseDown = () => {
-    longPressActive = true;
-    longPressed = false;
-    longPressTimeout = setTimeout(function () {
-      if (longPressActive === true) {
-        longPressed = true;
+  const handleMouseDown = (location, idx) => {
+    longPressed.current = false;
+    setTimeout(function () {
+      longPressed.current = true;
+      if (location.type === SENSOR && longPressed.current) {
+        setIsSensor(true);
+        setSensorIdx(idx);
       }
-    }, 200);
+    }, 800);
```

Comment:
I think it could be nice to create a constant for the time so you don't have to change in 2 places.

Reply:
That's a good idea!